src/sidecar/sidecarManager.ts

```diff
@@ -520,11 +522,12 @@ export class SidecarManager {
           });
         }
       }
-      sidecarOutputChannel.appendLine(line);
+
+      appendSidecarLogToOutputChannel(line);
     });
 
     this.logTailer.on("error", (data: any) => {
-      sidecarOutputChannel.appendLine(`Error: ${data.toString()}`);
+      sidecarOutputChannel.error(`Error: ${data.toString()}`);
```

Comment:
While I'm here, should we mention that this is a vscode specific error message rather than a sidecar originated error?

Reply:
Yeah that's one thing that isn't great about using the `LogOutputChannel` -- it's going to treat them all as log lines, so we'd have to come up with our own sort of prefix to show the error was extension-side. 🤔

Reply:
Yeah, that's fine I guess -- something like `[VS Code internal]` maybe.

Reply:
Or actually, `Failed to read sidecar logs, error: %s`

Reply:
Another option is just sending that to the main `Confluent` output channel instead of the `Confluent (Sidecar)` output channel, which may be more straightforward

Reply:
Ohh yeah that would be nice.